hooks/nvidia-device-plugin/Makefile

```diff
@@ -19,10 +19,10 @@ all: image
 .PHONY: image push
 
 image:  ## Build the image
-	docker build -t dcwangmit01/nvidia-device-plugin:0.1.0 -f image/Dockerfile image/
+	docker build --network host -t fifarzh/nvidia-device-plugin:0.2.0-cuda10.0 -f image/Dockerfile image/
```

Comment:
I'm wondering if it would be easier to make docker registry a required variable in the Makefile, something like:

```make
.PHONY: image push check-docker-registry

check-docker-registry:
ifndef DOCKER_REGISTRY
	$(error DOCKER_REGISTRY is undefined)
endif

image: check-docker-registry  ## Build the image
	docker build --network host -t $(DOCKER_REGISTRY)/nvidia-device-plugin:0.2.0-cuda10.0 -f image/Dockerfile image/

push: check-docker-registry   ## Push the image
	docker push $(DOCKER_REGISTRY)/nvidia-device-plugin:0.2.0-cuda10.0
```

This way users are required to build and host the image themselves rather than relying on a contributor-hosted image.  The README.md below could mention the need to provide a `DOCKER_REGISTRY` variable to the make commands and the hook example would have

```yaml
  hooks:
   - execContainer:
       image: DOCKER_REGISTRY/nvidia-device-plugin:0.2.0-cuda10.0 # Replace DOCKER_REGISTRY with the registry used to host the image
```

thoughts?

Reply:
much better, testing the new Makefile